package.json

```diff
@@ -23,7 +25,7 @@
     "postcss": "^7.0.27",
     "postcss-preset-env": "^6.7.0",
     "rollup": "^1.32.1",
-    "rollup-plugin-babel": "^4.4.0",
+    "@rollup/plugin-babel": "^5.1.0",
```

Comment:
Did you install this with `pnpm i @rollup/plugin-babel`? I'm curious why `pnpm-lock.yaml` wasn't updated. (Same with jsdoc)

Reply:
No, I used npm install - For some reason pnpm hangs for me after this:

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /Users/marcus/.pnpm-store/v3
  Virtual store is at:             node_modules/.pnpm
Resolving: total 949, reused 941, downloaded 8, done
node_modules/.pnpm/core-js@3.6.4/node_modules/core-js: Running postinstall script, done in 84ms
node_modules/.pnpm/node-sass@4.13.1/node_modules/node-sass: Running install script, done in 992ms
node_modules/.pnpm/node-sass@4.13.1/node_modules/node-sass: Running postinstall script...

Reply:
btw, I'm also seeing this warning when I try to run pnpm
```The "shamefully-flatten" setting has been renamed to "shamefully-hoist". Also, in most cases you won't need "shamefully-hoist". Since v4, a semistrict node_modules structure is on by default (via hoist-pattern=[*]).```

Reply:
Maybe pnpm hangs because you've both used npm and pnpm? The shamefully-hoist warning is fine. Cannot remove "shamefully-hoist" from `.npmrc` just yet, but hopefully in the future. Hopefully all of this will go away once we all use https://deno.land/ ;)

Reply:
I would do `rm -rf node_modules/` and try `pnpm i` again.

Reply:
Found a way to get more verbose output now, and it seems it's just spending suprising amounts of time compiling node-sass, not actually hanging. it actually finished after some minutes

Reply:
force pushed a squash to include updated lockfile and fix typo.